--- ./cache/openexr-openexr-d847d1e/IlmBase/IlmThread/CMakeLists.txt	2012-07-26 11:36:12.000000000 +0800
+++ ./001_IlmBase_IlmThread_CMakeLists.txt	2012-07-31 14:17:21.000000000 +0800
@@ -1,5 +1,7 @@
 # yue.nicholas@gmail.com
 
+INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
+
 IF (WIN32)
   SET( ILMTHREAD_LIBRARY_SOURCES
     IlmThread.cpp
@@ -40,10 +42,12 @@
 )
 
 INSTALL ( FILES 
-  IlmThreadPool.h
   IlmThread.h
-  IlmThreadSemaphore.h
+  IlmThreadForward.h
   IlmThreadMutex.h
+  IlmThreadNamespace.h
+  IlmThreadPool.h
+  IlmThreadSemaphore.h
   DESTINATION
   include/OpenEXR
 )
