--- ./build/imaging/ilmbase/2.0-beta1/openexr-openexr-27b2bf9/IlmBase/Iex/CMakeLists.txt	2012-08-02 15:41:52.000000000 +0800
+++ ./patches/ilmbase/2.0-beta1/001[-]IlmBase[-]Iex[-]CMakeLists.txt	2012-08-27 14:06:29.000000000 +0800
@@ -12,12 +12,18 @@
 )
 
 INSTALL ( FILES 
+  Iex.h
   IexBaseExc.h
-  IexMathExc.h
-  IexThrowErrnoExc.h
   IexErrnoExc.h
+  IexExport.h
+  IexForward.h
   IexMacros.h
-  Iex.h
+  IexMathExc.h
+  ../IexMath/IexMathFloatExc.h
+  ../IexMath/IexMathFpu.h
+  ../IexMath/IexMathIeeeExc.h
+  IexNamespace.h
+  IexThrowErrnoExc.h
   DESTINATION
   include/OpenEXR
 )
