--- ./build/imaging/field3d/1.3.2/CMakeLists.txt	2012-08-20 17:12:00.000000000 +0800
+++ ./CMakeLists.txt	2012-08-21 15:35:24.000000000 +0800
@@ -43,9 +43,9 @@
 # f3dinfo relies on program_options but don't include it, since
 # for some reason, unlike all the other boost components, a link is
 # forced via a pragma.
-FIND_PACKAGE (Boost COMPONENTS regex thread)
+FIND_PACKAGE (Boost COMPONENTS regex thread system)
 ELSE ()
-FIND_PACKAGE (Boost COMPONENTS regex thread program_options)
+FIND_PACKAGE (Boost COMPONENTS regex thread program_options system)
 FIND_PACKAGE (MPI)
 ENDIF ()
 
@@ -112,7 +112,7 @@
   LIST ( APPEND Field3D_Libraries_Shared
     Iex Half IlmThread Imath
     pthread dl z )
-  SET ( Field3D_DSO_Libraries ${Field3D_Libraries_Shared} )
+  SET ( Field3D_DSO_Libraries ${Field3D_Libraries_Shared} ${Boost_LIBRARIES} )
   SET ( Field3D_BIN_Libraries Field3D ${Field3D_Libraries_Shared}
         ${Boost_LIBRARIES} )
 ENDIF ( )
@@ -123,7 +123,7 @@
       optimized ${lib}
       debug ${lib}_d )
   ENDFOREACH()
-  SET ( Field3D_DSO_Libraries ${Field3D_Libraries_Shared} )
+  SET ( Field3D_DSO_Libraries ${Field3D_Libraries_Shared} ${Boost_LIBRARIES} )
   SET ( Field3D_BIN_Libraries Field3D ${Boost_LIBRARIES} )
 ENDIF ()
 
