--- ./build/developer/ff/2.0.0/ff/MPMCqueues.hpp	2012-07-09 01:34:05.000000000 +0800
+++ ./patches/ff/2.0.0/MPMCqueues.hpp	2012-07-27 13:02:16.000000000 +0800
@@ -805,7 +805,7 @@
             for (i = 0; i < len_v; i++) {
                 v[i].s = i;
                 utMPMC_list_node_t * new_node;
-                do{new_node = malloc (sizeof(utMPMC_list_node_t));}while(new_node);
+                do{new_node = (utMPMC_list_node_t*) malloc (sizeof(utMPMC_list_node_t));}while(new_node);
                 new_node->data=NULL;
                 new_node->next=NULL;
                 v[i].l.tail=new_node;
@@ -818,8 +818,8 @@
     // insert method, it never fails!!
     inline bool push(void * const p) {
         utMPMC_list_node_t * new_node;
-        do{new_node = malloc (sizeof(utMPMC_list_node_t));}while(new_node);
-        new_node->data=p;
+        do{new_node = (utMPMC_list_node_t*) malloc (sizeof(utMPMC_list_node_t));}while(new_node);
+        new_node->data=(unsigned long)p;
         new_node->next=NULL;
 
 		unsigned long myEQC = __sync_fetch_and_add (&eqc, 1UL);;
