--- ./build/developer/ff/2.0.0/tests/CMakeLists.txt	2011-08-24 01:38:40.000000000 +0800
+++ ./patches/ff/2.0.0/osx/CMakeLists.txt	2012-07-27 12:54:44.000000000 +0800
@@ -31,7 +31,7 @@
 if (NOT(GCC_TOO_OLD))
   add_executable( test_mpmc test_mpmc.cpp)
   if (NOT(CMAKE_SYSTEM_NAME MATCHES "Windows"))
-    SET_TARGET_PROPERTIES(test_mpmc PROPERTIES COMPILE_FLAGS " -Wno-strict-aliasing -O6  -finline-functions ")
+    SET_TARGET_PROPERTIES(test_mpmc PROPERTIES COMPILE_FLAGS " -Wno-strict-aliasing -O3  -finline-functions ")
   endif (NOT(CMAKE_SYSTEM_NAME MATCHES "Windows"))
 endif (NOT(GCC_TOO_OLD))
  
