--- blob.hpp.orig	2012-11-09 13:13:57.000000000 +0800
+++ blob.hpp	2012-11-09 13:23:50.000000000 +0800
@@ -424,7 +424,9 @@
 
 			fd = open(path.c_str(), O_RDONLY);
 			if (fd >= 0) {
+#ifdef __linux__
 				posix_fadvise(fd, 0, 0, POSIX_FADV_DONTNEED);
+#endif
 				close(fd);
 			}
 		}
@@ -589,7 +591,7 @@
 			if (idx != -1) {
 				m_chunk_idx = idx;
 				fin_t in;
-				m_files[idx]->read_index<fin_t>(in, m_chunks, m_chunks_unsorted, 0);
+				m_files[idx]->template read_index<fin_t>(in, m_chunks, m_chunks_unsorted, 0);
 
 				log(SMACK_LOG_INFO, "%s: read-index: idx: %d, sorted: %zd, unsorted: %zd, num: %zd\n",
 						m_path.c_str(), idx, m_chunks.size(), m_chunks_unsorted.size(), this->num());
